File: code/RLUD.java
import java.util.*;
public class Main{
    public static void main(String args[]){
        String[] arr ={"R","R","R","U","D","D"};
        int[] cur ={1,1};
        int i=0;
        int n=5;
        while(i<arr.length){
            int[] add =cur.clone();
            if(arr[i].equals("R")) add[1] +=1;
            if(arr[i].equals("L")) add[1] -=1;
            if(arr[i].equals("U")) add[0] -=1;
            if(arr[i].equals("D")) add[0] +=1;
            if(add[0]>=1 && add[0]<=n && add[1]>=1 && add[1]<=n){
                cur =add.clone();
            }
            i+=1;
        }
        System.out.println(Arrays.toString(cur));
    }
}

File: code/RLUD.py
arr=["R","R","R","U","D","D"]
cur=[1,1]
i =0
n=5
while i<len(arr):
    add =cur.copy()
    if arr[i] =='R': add[1]+=1
    if arr[i] =='L': add[1]-=1
    if arr[i] =='U': add[0]-=1
    if arr[i] =='D': add[0]+=1
    if 1<=add[0]<=n and 1<=add[1]<=n:
        cur =add.copy()
    i+=1
print(cur)
    
